src/test/cls_refcount/test_cls_refcount.cc

```diff
@@ -133,7 +133,6 @@ TEST(cls_rgw, test_put_snap) {
   ASSERT_EQ(0, ioctx.snap_create("snapbar"));
 
   librados::ObjectWriteOperation *op = new_op();
-  op->create(false);
```

Comment:
do we only get ENODATA if the op touches the object first? seems like we'd want runs through rgw/rbd/fs suites as well to make sure this doesn't affect them